### PR TITLE
Enable codecov coverage tracking

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,3 +12,5 @@ git:
 #script: # use the default script which is equivalent to the following
 # - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
 # - julia -e 'Pkg.clone(pwd()); Pkg.build("PkgDev"); Pkg.test("PkgDev"; coverage=true)'
+after_success:
+  - julia -e 'cd(Pkg.dir("MyPkg")); Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
```
Comment (ararslan) [inline .travis.yml]: MyPkg?

**Comment by kshyatt** [comment]
Ok, now that I fixed this, can we merge?

**Comment by ararslan** [comment]
There's something weird going on with AppVeyor, not really sure what the deal is. It looks like it's trying to do coverage stuff on AV? Shouldn't be...

**Comment by tkelman** [comment]
It's running PkgDev's own tests, which used to work I think, something in nightly may have broken them? They aren't running on 0.5 release yet apparently.

**Pull request merged**